docs/source/multimodal/speech_llm/intro.rst

```diff
@@ -21,19 +23,20 @@ Model Architecture
 ^^^^^^^^^^^^^^^^^^
 
 One way to incorporate speech into LLM is to concatenate speech features with the token embeddings of the input text prompt before being fed into the LLM. In this way, the LLM can have direct access to the speech information when generating the output text.
-    .. image:: https://github.com/NVIDIA/NeMo/releases/download/v1.23.0/salm.png
-        :align: center
-        :alt: SALM model
-        :scale: 50%
+
+.. image:: https://github.com/NVIDIA/NeMo/releases/download/v1.23.0/salm.png
+    :align: center
+    :alt: SALM model
+    :scale: 50%
 
 
 
 Another way is to use cross-attention mechanism, by using text embeddings to attend to speech embeddings to extract task-specific information from the speech embeddings. In order to minimize the computational cost of cross-attention, we add a cross-attention module only before the LLM.
```

Comment:
revise.

Another approach is to use a cross-attention mechanism, where text embeddings attend to speech embeddings to extract task-specific information. To minimize the computational cost, we add a cross-attention module only before the LLM.